sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -503,39 +504,59 @@ export class StudentService extends RecordDataModelService<Student> {
    * Use the information available in the authentication token to update
    * the user and student data currently on DB.
    * @param studentToken student authentication token.
+   * @param options method options:
+   * - `userId` user id provided in case the student data
+   * is updated by the ministry.
    * @returns updated student, if some data was changed.
    */
-  async synchronizeFromUserToken(
-    studentToken: StudentUserToken,
+  async updateStudentUserData(
```

Comment:
The comment does not make sense after this change because the information does not come from the token only.
"Use the information available in the authentication token"